### PR TITLE
Fixes #12: Adds Travis CI to check builds

### LINKED ISSUE #12: Add TravisCI
Add TravisCI to the project to check build failures.

### PR DESCRIPTION
**Pull request by SKrPl**
Solves issue: https://github.com/loklak/loklak_jlib_api/issues/12

### CODE REVIEW

**Comment by SKrPl** [comment]
@Achint08 @djmgit @hemantjadon @kavithaenair @SarishiNoHara @singhpratyush @vibhcool Please review this.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: java
+
+jdk: oraclejdk8
+
+before_script:
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```
Comment (vibhcool) [inline .travis.yml:6]: I have a doubt, `-DskipTests` command's task is done by `-Dmaven.javadoc.skip=true`, doesn't it? 
see this https://www.mkyong.com/maven/how-to-skip-maven-unit-test/

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+script:
+  - mvn test -B
```
Comment (vibhcool) [inline .travis.yml:9]: and then why this command?

File: .travis.yml (hunk 1 above)
Comment (SKrPl) [reply .travis.yml:6]: This particular command installs the dependencies of the project. So, tests are skipped and Javadoc generation is skipped at that time. So, both commands have a different use.
Reference: TravisCI documentation: [link](https://docs.travis-ci.com/user/languages/java/#Projects-Using-Maven)

File: .travis.yml (hunk 2 above)
Comment (SKrPl) [reply .travis.yml:9]: Once dependencies are installed, it compiles the unit-tests and runs them to check for build failures.

**Review by vibhcool — APPROVED** [review]

**Pull request merged**